<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Resource Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="4">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
      <property name="icon">
       <iconset resource="../SABUtils/resources/SABUtils.qrc">
        <normaloff>:/SABUtilsResources/add.png</normaloff>:/SABUtilsResources/add.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="3" colspan="4">
     <widget class="QTreeWidget" name="files">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Language</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Alias</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Compression</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Compression Level</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Compression Threshold</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="removeBtn">
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="icon">
       <iconset resource="../SABUtils/resources/SABUtils.qrc">
        <normaloff>:/SABUtilsResources/delete.png</normaloff>:/SABUtilsResources/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QGroupBox" name="properties">
      <property name="title">
       <string>Properties</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="aliasLabel">
         <property name="text">
          <string>Alias:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="alias"/>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QGroupBox" name="compressionEnabled">
         <property name="title">
          <string>Compression (Unchecked uses defaults)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QComboBox" name="compression">
            <item>
             <property name="text">
              <string>Best</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>zstd</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>zlib</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="threshold">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
            <property name="value">
             <number>70</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Threshhold:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Algorithm:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Level:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="level">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>19</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="resourcePathLabel">
         <property name="text">
          <string>Resource Path:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="resourcePath">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="resourceURLLabel">
         <property name="text">
          <string>Resource URL:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="resourceURL">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="languageLabel">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="prefixLabel">
         <property name="text">
          <string>Prefix:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="prefix"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="language"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string>Filename:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="fileName">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionAddFiles"/>
    <addaction name="actionAddPrefix"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../SABUtils/resources/SABUtils.qrc">
     <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../SABUtils/resources/SABUtils.qrc">
     <normaloff>:/SABUtilsResources/load.png</normaloff>:/SABUtilsResources/load.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="text">
    <string>Add Files...</string>
   </property>
  </action>
  <action name="actionAddPrefix">
   <property name="text">
    <string>Add Prefix...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>files</tabstop>
  <tabstop>prefix</tabstop>
  <tabstop>language</tabstop>
  <tabstop>fileName</tabstop>
  <tabstop>alias</tabstop>
  <tabstop>compressionEnabled</tabstop>
  <tabstop>compression</tabstop>
  <tabstop>level</tabstop>
  <tabstop>threshold</tabstop>
  <tabstop>resourcePath</tabstop>
  <tabstop>resourceURL</tabstop>
 </tabstops>
 <resources>
  <include location="resources/application.qrc"/>
  <include location="../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
